server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java

```diff
@@ -89,6 +112,21 @@ private void addServices(ServerBuilder sb) {
         new IcebergRestCatalogService(catalogService, schemaService, tableService, metadataService),
         icebergRequestConverter,
         icebergResponseConverter);
+
+    ServerPropertiesUtils serverPropertiesUtils = ServerPropertiesUtils.getInstance();
+    String authorization = serverPropertiesUtils.getProperty("server.authorization");
+    // TODO: eventually might want to make this secure-by-default.
+    if (authorization != null && authorization.equalsIgnoreCase("enable")) {
+      LOGGER.info("Authorization enabled.");
+      AuthDecorator authDecorator = new AuthDecorator();
+      ExceptionHandlingDecorator exceptionDecorator =
+          new ExceptionHandlingDecorator(new GlobalExceptionHandler());
+      sb.routeDecorator()
+          .pathPrefix(basePath)
+          .exclude(basePath + "auth/tokens")
```

Comment:
Isn't this `controlPath + "auth/tokens"`?

Reply:
good catch. I recently moved the endpoint out of the formal `/unity-catalog` namespace, but missed this.